package.json

```diff
@@ -63,6 +65,7 @@
     "pg": "^8.4.1",
     "postgraphile": "^4.8.0",
     "postgraphile-plugin-connection-filter": "^2.0.0",
+    "prettier": "^2.2.1",
```

Comment:
Hmmm, don't we already have Prettier installed globally? We're definitely already using it in our project (we have a .prettierrc file). Is this going to cause a problem, or do we need it?
Or should this local version be the one have active for formatting?

Reply:
I think it comes with extension, I've changed to use npx:

6f9de176bb56d9e96e84dc132a679a3a6a08f9d8